### DISCUSSION TITLE
relationship list

### DISCUSSION THREAD

**Discussion by rafikApp** [Budibase/budibase#9456] [Help]
Hi everyone
I have a relational list to let the user choose an option, and I have a text field to add the option not found in the list to the database.
In case the text added by the user already exists in the list (i.e. the database) I want to select it directly in the list to avoid duplicate lines in the database.
for this I provided the text field with a data provider (filter == value entered by the user) if the table is not empty then it means that the value already exists in the list.
so I have to select it from the list for that I made the following instructions.

![1](https://user-images.githubusercontent.com/106697208/215003450-4654aa3b-5d85-4fa3-b6a3-a76e9cb64c9a.PNG)
![2](https://user-images.githubusercontent.com/106697208/215003455-6661832f-677f-4d3b-82ae-1eb056b2dfcb.PNG)
![3](https://user-images.githubusercontent.com/106697208/215003458-fcbe46a7-7e7e-478a-a567-085e3c4a65c9.PNG)

but it doesn't work, there's always a lag.

https://user-images.githubusercontent.com/106697208/215003971-1afc7da4-91ba-47cb-947d-77061a73e810.mp4

@melohagan

**Comment by melohagan** [comment]
Hey @rafikApp 

In your last screenshot you haven't selected a field?

**Reply by rafikApp** [reply]
Hey @melohagan 
No I selected a field, but I hid it.
